setup.cfg

```diff
@@ -14,9 +14,10 @@ classifiers =
 [options]
 packages = ice
 install_requires =
-    Faker
     defopt
+    faker
```

Comment:
Is this case-insensitive?

Reply:
Yes, pip and PyPI are case insensitive, and also -/_ are interchangeable.